bridge/qb/shared/main.lua

```diff
@@ -85,13 +87,15 @@ qbShared.CommaValue = lib.math.groupdigits
 qbShared.RandomStr = function(length)
     if length <= 0 then return '' end
     local pattern = math.random(2) == 1 and 'a' or 'A'
-    return qbShared.RandomStr(length - 1) .. lib.string.random(pattern)
+
+    return lib.string.random(pattern, length)
 end
 
 ---@deprecated use lib.string.random from ox_lib
 qbShared.RandomInt = function(length)
     if length <= 0 then return '' end
-    return qbShared.RandomInt(length - 1) .. lib.string.random('1')
+
+    return lib.string.random('.', length)
```

Comment:
Ditto, and you should be using `1` for the pattern, since `.` is both random letters and numbers, while `1` is only random numbers.